src/grt/src/GlobalRouter.tcl

```diff
@@ -282,6 +282,27 @@ proc write_guides { args } {
   grt::write_guides $file_name
 }
 
+sta::define_cmd_args "draw_route_guides" { nets }
+
+proc draw_route_guides { nets } {
+  set block [ord::get_db_block]
+  if { $block == "NULL" } {
+    utl::error GRT 223 "Missing dbBlock."
+  }
+
+  foreach net [get_nets $nets] {
+    if { $net != "NULL" } {
+      grt::highlight_net_route [sta::sta_to_db_net $net]
+    }
+  }
+}
+
+sta::define_cmd_args "erase_route_guides" {}
```

Comment:
I would change draw_route_guides {} (ie, empty list) to "erase" them instead of adding another command